traffic_ops/app/t/api/1.2/config_files_apache_traffic_server.t

```diff
@@ -72,7 +72,7 @@ my $count_response = sub {
 	return $t->success( is( scalar(@$r), $count ) );
 };
 
-$t->get_ok('/api/1.2/deliveryservices/list?logsEnabled=true')->status_is(200)->$count_response(2)
+$t->get_ok('/api/1.2/deliveryservices/list?logsEnabled=true')->status_is(200)->$count_response(3)
```

Comment:
Actualy it is related. I added a DS with the tenant, and the DS has "logsEnabled". So the count went fom 2 to 3